docs/2.7.0/docs/daml-finance/concepts/lifecycling.rst

```diff
@@ -187,7 +187,22 @@ assume that a new version of the instrument has already been created (as is the
 the new version of the instrument as part of its implementation. The latter can be useful if
 information required to create the new version is only available upon processing of the event, as is
 the case for :ref:`Generic Instrument
-<type-daml-finance-interface-instrument-generic-instrument-instrument-11652>` evolution.
+<type-daml-finance-interface-instrument-generic-instrument-instrument-11652>` evolution, as well as
+other Contingent Claims based instruments.
```

Comment:
Do we ever define what is a contingent claims based instruments prior to this point?